apps/docs/pages/guides/getting-started/tutorials/with-expo-react-native.mdx

```diff
@@ -169,10 +169,22 @@ Users would be able to sign in with their email and password.
 
 ```tsx components/Auth.tsx
 import React, { useState } from 'react'
-import { Alert, StyleSheet, View } from 'react-native'
+import { Alert, StyleSheet, View, AppState } from 'react-native'
 import { supabase } from '../lib/supabase'
 import { Button, Input } from 'react-native-elements'
 
+// Tells Supabase Auth to continuously refresh the session in the background if
+// the app is focused. When this is added, you will continue to receive
+// `onAuthStateChange` events with the `TOKEN_REFRESHED` or `SIGNED_OUT` event
+// if the user's session is terminated. This should only be registered once.
```

Comment:
The term `background` might confuse the reader as it's often used for mobile app life cycle. 
```suggestion
// Tells Supabase Auth to continuously refresh the session automatically if
// the app is in the foreground. When this is added, you will continue to receive
// `onAuthStateChange` events with the `TOKEN_REFRESHED` or `SIGNED_OUT` event
// if the user's session is terminated. This should only be registered once.
```

Also, not sure if the following line is adding any new information here. 

> When this is added, you will continue to receive`onAuthStateChange` events with the `TOKEN_REFRESHED` or `SIGNED_OUT` event